IPython/core/tests/test_ultratb.py

```diff
@@ -196,7 +196,7 @@ def test_non_syntaxerror(self):
             raise ValueError('QWERTY')
         except ValueError:
             with tt.AssertPrints('QWERTY'):
-                ip.showsyntaxerror()
+                ip.showtraceback()
```

Comment:
From the comment in this test, changing this misses the point of what it's supposed to be testing.